FIPS/pom.xml

```diff
@@ -38,17 +38,17 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <arrow.version>10.0.1</arrow.version>
-    <slf4j.version>1.7.36</slf4j.version>
+    <slf4j.version>2.0.6</slf4j.version>
     <jsoup.version>1.15.3</jsoup.version>
     <tika.version>2.4.1</tika.version>
     <jackson.version>2.14.2</jackson.version>
-    <jacksondatabind.version>2.13.4.2</jacksondatabind.version>
-    <httpclient.version>4.5.11</httpclient.version>
+    <httpclient.version>4.5.14</httpclient.version>
     <jacoco.version>0.8.4</jacoco.version>
     <jacoco.skip.instrument>true</jacoco.skip.instrument>
     <awssdk.version>1.12.501</awssdk.version>
-    <google.api.client.version>1.33.2</google.api.client.version>
-    <google.http.client.version>1.43.2</google.http.client.version>
+    <google.api.client.version>2.2.0</google.api.client.version>
+    <google.http.client.version>1.43.3</google.http.client.version>
+    <netty.version>4.1.100.Final</netty.version>
```

Comment:
why adding this? Idea is non FIPS jars should be role model for the FIPS jars. Ideally only difference should be Bouncy Castle jars. FIPS jar should have all dependency as non fips but not the otherway round.